processing/data_collection/gazette/spiders/sc_florianopolis.py

```diff
@@ -11,20 +11,24 @@
 
 class ScFlorianopolisSpider(BaseGazetteSpider):
     name = "sc_florianopolis"
-    URL = "http://www.pmf.sc.gov.br/governo/index.php?pagina=govdiariooficial"
     TERRITORY_ID = "4205407"
-    AVAILABLE_FROM = date(2015, 1, 1)  # actually from June/2009
+
+    start_date = date(2009, 6, 1)
 
     def start_requests(self):
-        target = date.today()
-        while target >= self.AVAILABLE_FROM:
-            year, month = str(target.year), str(target.month)
-            data = dict(ano=year, mes=month, passo="1", enviar="")
-            yield FormRequest(url=self.URL, formdata=data, callback=self.parse)
-            if hasattr(self, "start_date") and target <= self.start_date:
-                break
+        initial_date = self.start_date
```

Comment:
Why creating this additional variable? Why not use the `start_date` directly?

Reply:
For no good reason actually. Removed it.